Site updates

cnunciato opened the pull request.

joeduffy commented on content/_index.md:
```diff
@@ -25,16 +24,16 @@ carousel:
             any .NET language, including C#, F#, and VB.
         - title: Your favorite tools.
-          description: Use familiar IDEs, test frameworks, and tools.
+          description: Use familiar IDEs, test frameworks, and tools. Resource mocking for easy verification.
```
```suggestion
          description: Use familiar IDEs, test frameworks, and tools.
```

We already say test frameworks -- doesn't feel like we need to complicate the point with the mocking sentence.

isaac-pulumi replied:
I'm okay with this edit.

joeduffy commented on content/_index.md:
```diff
@@ -25,16 +24,16 @@ carousel:
+          description: Use familiar IDEs, test frameworks, and tools. Resource mocking for easy verification.
         - title: Share and reuse.
-          description: Codify best practices and share them with your team or a growing community of practitioners.
+          description: Codify best practices and policies then share them with your team or community as self-service architectures.
```
Hm, I don't quite follow the "self-service architectures" point. What are we trying to say?

isaac-pulumi replied:
I think we've drifted off of the messaging - can we snap to the descriptions here on the Press Release and let me know if a shorter version is needed to fit the space?

isaac-pulumi replied:
Link to PR in slack

joeduffy replied:
I'm still having trouble parsing this. What does "sharing ... as self-service architectures" mean?

isaac-pulumi replied:
This should have been addressed in the last round of edits. Not sure where this language came from but it's gone in the latest preview...

joeduffy commented on layouts/partials/alert.html:
```diff
@@ -1,10 +1,16 @@
+            <span class="flex rounded-full bg-orange-700 border-2 border-orange-600 uppercase text-white px-2 py-1 text-xs font-bold mr-2">New</span>
             <span class="text-left flex-auto text-white text-sm md:text-base leading-normal">
-                Join us on April 29 for a fun walk-through of the latest Pulumi features and capabilities with Corey Quinn.
+                Join us on April 29 for a fun walk-through of the latest Pulumi features with Corey Quinn.
```
```suggestion
                Pulumi 2.0 is available! Learn more here.
```

The CTA here should be to learn more about and use the product -- if they join the webinar, great, but that too's purpose is to drive people to learn more about and use the product.

joeduffy replied:
Ah, also the link should go to /superpowers. It seem to go to the webinar page right now.

joeduffy commented:
One layout I had imagined, which I am curious what you think about is, to position the new mascot just to the right and at a peer level to the main hero tagline/CTA.

A bit like this (but without any of the other wording changes shown here):

![image](https://user-images.githubusercontent.com/3953235/79485390-00194980-7fca-11ea-97a0-74abc656b532.png)

That feels better to me for a few reasons:

* Draws the eye more to the CTA, rather than away from it.
* He's leaning into the tagline, rather than seeming to run away from it.
* The prominent sparkles align nicely to the superpowers point in the text.
* The mascot is a permanent part of our new branding, not a temporary part of the alert.

Thoughts?

joeduffy commented on content/_index.md:
```diff
@@ -59,24 +58,20 @@ carousel:
 quotes:
```
Did we plan on changing the logos on the homepage to link to the case studies when available?

cnunciato replied:
No reason not to! Will do.

joeduffy replied:
Thanks for changing this! My only suggestion now is to somehow visually indicate which one has a case study versus not. It very well may be that we want to move these to a completely different card structure -- with a teaser about what the case study concludes. Meantime, maybe just a badge of some sort?

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                <div class="md:ml-8 md:mt-8 md:w-3/4">
+                    <h1 class="text-4xl">Modern Infrastructure with <span class="text-blue-300 font-semibold">Superpowers</span>.</h1>
+                    <p class="text-xl pr-48 text-purple-100 leading-relaxed">
+                        We're unleashing new capabilities for Delivery, Provisioning,
```
```suggestion
                        Pulumi 2.0 includes new capabilities for Delivery, Provisioning,
```

One thing to keep in mind is that the features are available as soon as this page goes live. We don't need to make it look like they're somehow held back until after the webinar.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                    <p class="text-xl pr-48 text-purple-100 leading-relaxed">
+                        We're unleashing new capabilities for Delivery, Provisioning,
+                        Architecture, Testing and Policy-as-code &mdash; on any cloud, using your
+                        favorite languages. Join us as we walk through our favorite new features.
```
What if a user wants to get started right away? (The ideal case!) "Join us" implies they can't do that yet?

cnunciato replied:
I'll adjust the wording here to be more immediately inviting to what's on the page now.

isaac-pulumi replied:
> > Love it -- you mean on the home page, right? Or on the superpowers page?
> 
> I meant on the homepage, but possibly both would be nice 😄

Agree that some sparkle on /home would be great too.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0" >
+                            <a href="{{ relref . "/docs/get-started" }}">
```
These links are a bit subtle for some reason. It wasn't clear I could click until I hovered.

What I wondered is whether we should use buttons, since they're our standard CTA mechanism. For each superpower, there could be three:

LEARN | WATCH | START

* LEARN goes to the page we currently link to
* WATCH goes to the video (many of which are below)
* START is a normal get started CTA

joeduffy replied:
Related, I do think we need a landing page for provisioning that is better than just going to getting started. How Pulumi Works is the closest we have, but I do think we should spend some cycles spiffing that up. I've wanted to link to something like this several times recently for customers who want to understand how Pulumi works with previews, goal states, etc. Sadly that page isn't great, but it's the best we have -- maybe use that?

cnunciato replied:
That's a good point, and I can see it now. Will see what I can do to make this clearer.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Manage infrastructure with your favorite languages, libraries, and editors
+                            instead of wrestling with legacy formats. Preview changes and track
```
```suggestion
                            instead of wrestling with proprietary YAML dialects and DSLs. Preview changes and track
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Manage infrastructure with your favorite languages, libraries, and editors
+                            instead of wrestling with legacy formats. Preview changes and track
+                            updates on-prem and any cloud.
```
```suggestion
                            updates on any public, private, or hybrid cloud.
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Ensure that your infrastructure is correct before and after deployment.
+                            New resource mocking and test framework support make testing and
```
```suggestion
                            Use unit and integration tests, mocking, or ephemeral environments for quick and easy validation.
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Continuously ship application and infrastructure updates using common
+                            workflows. Integrates with your CI/CD toolchain to improve velocity and
```
```suggestion
                            workflows. Leverage your existing CI/CD toolchain to improve velocity and
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Continuously ship application and infrastructure updates using common
+                            workflows. Integrates with your CI/CD toolchain to improve velocity and
+                            visibility into infrastructure dependencies and state.
```
```suggestion
                            visibility into all of your team's cloud deployment activity.
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Stop reinventing the wheel with each new microservice component and create
```
```suggestion
                            Codify application and infrastructure patterns using components, and share them using package managers. Leverage an ecosystem of libraries with built-in best practices.
```

isaac-pulumi replied:
looks fine - will need to see in preview if the extra text causes issues.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 md:border-b-0 leading-relaxed">
+                            Enforce standards for deployments with auditing capabilities across roles,
```
Enforce security, compliance, and cost controls to prevent mistakes from reaching production. Define custom policies using your favorite language and flexibly apply them across your projects.

isaac-pulumi replied:
looks fine - will need to see in preview if the extra text causes issues.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "/docs/get-started" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Any cloud, any language</span>
```
This feels a bit odd to be a superpower.

isaac-pulumi replied:
Two thoughts here:

1. We made a big deal in the roadmap about multi-language support and language talks feature prominently, but, we kind of leave this out of 'superpowers'
2. I actually added this to balance out the 2x3 list on this page.  

Happy to keep or drop if this feels artificial.

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 leading-relaxed">
+                            Pulumi works with top programming languages across all major public clouds
+                            with support for on-prem, hybrid and multi-cloud deployments.
```
```suggestion
                            with support for over three dozen other infrastructure service providers, including private and hybrid clouds.
```

isaac-pulumi replied:
```suggestion
                            with support for dozens of popular infrastructure service providers including private and hybrid clouds.
```

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,276 @@
+{{ define "hero" }}
+    <header class="header-hero shadow-2xl relative overflow-hidden">
```
I feel like this page would benefit from one of these nav banners that we use on other pages, to jump to sections:

![image](https://user-images.githubusercontent.com/3953235/79487164-a9f9d580-7fcc-11ea-9f2d-89eccc7e8d4d.png)

If we did this, I would imagine the navigation links being something like SUPERPOWERS, LAUNCH EVENT, WATCH THE TALKS.

isaac-pulumi replied:
Agreed - advancing to each section will make stuff below the fold more discoverable.

isaac-pulumi commented:
> One layout I had imagined, which I am curious what you think about is, to position the new mascot just to the right and at a peer level to the main hero tagline/CTA.
> 
> A bit like this (but without any of the other wording changes shown here):
> 
> ![image](https://user-images.githubusercontent.com/3953235/79485390-00194980-7fca-11ea-97a0-74abc656b532.png)
> 
> That feels better to me for a few reasons:
> 
> * Draws the eye more to the CTA, rather than away from it.
> * He's leaning into the tagline, rather than seeming to run away from it.
> * The prominent sparkles align nicely to the superpowers point in the text.
> * The mascot is a permanent part of our new branding, not a temporary part of the alert.
> 
> Thoughts?

**I like this idea.** Let's do this.

cnunciato commented:
> > One layout I had imagined, which I am curious what you think about is, to position the new mascot just to the right and at a peer level to the main hero tagline/CTA.
> > A bit like this (but without any of the other wording changes shown here):
> > ![image](https://user-images.githubusercontent.com/3953235/79485390-00194980-7fca-11ea-97a0-74abc656b532.png)
> > That feels better to me for a few reasons:
> > 
> > * Draws the eye more to the CTA, rather than away from it.
> > * He's leaning into the tagline, rather than seeming to run away from it.
> > * The prominent sparkles align nicely to the superpowers point in the text.
> > * The mascot is a permanent part of our new branding, not a temporary part of the alert.
> > 
> > Thoughts?
> 
> **I like this idea.** Let's do this.

I do too. I had been thinking of the mascot's placement here as temporary and associated with launch, so I'm glad you pointed that out. Totally makes sense to reposition it in this way, then. Will do.

Thank you for the feedback, folks. Much appreciated.

zchase commented:
@cnunciato The links for the tech talks are as follows:
Policy as Code for Any Cloud: `/webinars/getting-started-policy-as-code/`
Managing Any Cloud with .NET: `/webinars/manage-any-cloud-dotnet/`
Building Self-service Clouds with Pulumi: `/webinars/building-self-service-clouds/`
How to Test Cloud Infrastructure with Pulumi: `/webinars/how-to-test-cloud-infrastructure/`

These links will be present once this PR is merged: https://github.com/pulumi/docs/pull/2974

joeduffy commented:
One random idea (not trying to create crazy amounts of work, so take it as simply an idea). What if, when you landed on the page, the word "superpowers" (in the "Now with superpowers." blurb) animated ever so slightly with a bit of sparkles around it. Your existing sparkle animation made me wonder...

cnunciato commented:
> What if, when you landed on the page, the word "superpowers" (in the "Now with superpowers." blurb) animated ever so slightly with a bit of sparkles around it. 

Love it -- you mean on the home page, right? Or on the superpowers page?

cnunciato commented:
@joeduffy @isaac-pulumi Ok, I've addressed all feedback with these exceptions:

* Links to the talks themselves don't work yet (per @zchase's note above) -- they'll be there soon
* Sparkle, maybe, to the Superpowers word in /superpowers

I think that's it. So if you see anything else at this point (once current Pulumify run completes), do let me know here, and I'll get this wrapped up tomorrow.

joeduffy commented:
> Love it -- you mean on the home page, right? Or on the superpowers page?


I meant on the homepage, but possibly both would be nice 😄

joeduffy commented on layouts/page/superpowers.html:
```diff
@@ -0,0 +1,404 @@
+                        Get Started
+                    </a>
+                    <a class="btn btn-lg btn-orange-translucent" href="#launch">
+                        Watch the Launch Event
```
Until the event is live, it feels like this is a slight bait-and-switch on the CTA. Should it say "Register for the Launch Event" for now? It's a bit more honest and at least you know right away that it's something in the future, rather than right now.

isaac-pulumi replied:
Maybe "Join the Launch Event"? Register sounds like a big commitment...

cnunciato replied:
Or Catch the Launch Event, or Join Us for Launch.

isaac-pulumi replied:
"Join Us for Launch" - nice and short
